src/meili/map_matcher_factory.cc

```diff
@@ -68,23 +68,28 @@ boost::property_tree::ptree MapMatcherFactory::MergeConfig(const Options& option
   }
 
   // Check for overrides of matcher related directions options. Override these values in config.
-  if (options.search_radius() && customizable.find("search_radius") != customizable.end()) {
+  if (options.has_search_radius() && customizable.find("search_radius") != customizable.end()) {
```

Comment:
these were bugs... we need to always check in protobuf if the value is there before using it. it might be masked by something upstream but ive fixed it anyway